scene/2d/camera_2d.cpp

```diff
@@ -801,7 +801,7 @@ void Camera2D::_bind_methods() {
 	ADD_PROPERTYI(PropertyInfo(Variant::INT, "limit_bottom", PROPERTY_HINT_NONE, "suffix:px"), "set_limit", "get_limit", SIDE_BOTTOM);
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "limit_smoothed"), "set_limit_smoothing_enabled", "is_limit_smoothing_enabled");
 
-	ADD_GROUP("Follow Smoothing", "follow_smoothing_");
+	ADD_GROUP("Follow Smoothing", "position_smoothing_");
```

Comment:
I think the group title should also match the properties:
```suggestion
	ADD_GROUP("Position Smoothing", "position_smoothing_");
```